phpstan.neon

```diff
@@ -50,6 +50,10 @@ parameters:
         # known type
         - '#Parameter \#1 \$className of method Rector\\Core\\Reflection\\ReflectionResolver\:\:resolveMethodReflection\(\) expects class\-string, string given#'
 
+        -
+            message: '#Call to an undefined method PHPStan\\Reflection\\ExtendedMethodReflection\:\:isAbstract\(\)#'
```

Comment:
This should be fixed with set actual object with `isAbstract()` method, you can debug actual object with `dump($variable)`

Reply:
The actual class is `PHPStan\Reflection\Php\PhpMethodReflection`  which has `isAbstract` but when i do something like `assert($extendedMethodReflection instanceof PHPStan\Reflection\Php\PhpMethodReflection);` i get error ` Class Rector\Symfony\Rector\ClassMethod\PHPStan\Reflection\Php\PhpMethodReflection not found. `

Reply:
You need to use FQCN, or that will refer current namespace:

```diff
-assert($extendedMethodReflection instanceof PHPStan\Reflection\Php\PhpMethodReflection); 
+assert($extendedMethodReflection instanceof \PHPStan\Reflection\Php\PhpMethodReflection); 
```

Reply:
Aah thanks :)